src/types/utils.ts

```diff
@@ -429,6 +348,25 @@ export function getABIEncoderVersion(
     return lt(compilerVersion, "0.8.0") ? ABIEncoderVersion.V1 : ABIEncoderVersion.V2;
 }
 
+export function isFunctionCallExternal(call: FunctionCall): boolean {
+    if (call.kind !== FunctionCallKind.FunctionCall) {
+        return false;
+    }
+
+    if (
+        call.vFunctionCallType === ExternalReferenceType.Builtin &&
+        CALL_BUILTINS.includes(call.vFunctionName)
+    ) {
+        return true;
+    }
+
+    if (call.vExpression.typeString.includes(FunctionVisibility.External)) {
```

Comment:
Can the type string be something like this `function (externalStructName memory) internal`? This might be fragile. Lets file an issue for a cleaner implementation of `isFunctionCallExternal`

Reply:
This is tracked by #171.